src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Extras/AppShell/BitAppShellDemo.razor.scss

```
@@ -0,0 +1,92 @@
+    border-radius: 0.375rem;
+    border: 1px solid var(--bit-clr-brd-sec);
+
+    &.short {
+        height: 9rem;
+    }
+}
+
+.page {
+    width: 100%;
+    display: flex;
+    // So a page shorter than the shell still fills it, which is what a sticky header and a sticky
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)